lib/Twig/NodeVisitor/Optimizer.php

```diff
@@ -45,6 +46,7 @@ public function __construct($optimizers = -1)
         }
 
         $this->optimizers = $optimizers;
+        $this->isPHP540RCorHigher = version_compare(phpversion(), '5.4.0RC1', '>=');
```

Comment:
Can you use `PHP_VERSION_ID >= 50400` instead? and perhaps rename the variable to just `$isPHP54`.